packages/router/src/operators/apply_redirects.ts

```diff
@@ -17,8 +17,8 @@ import {RouterConfigLoader} from '../router_config_loader';
 import {UrlSerializer} from '../url_tree';
 
 export function applyRedirects(
-    moduleInjector: Injector, configLoader: RouterConfigLoader, urlSerializer: UrlSerializer,
-    config: Routes): MonoTypeOperatorFunction<NavigationTransition> {
+    moduleInjector: EnvironmentInjector, configLoader: RouterConfigLoader,
```

Comment:
nit: should we rename an argument name as well?
```suggestion
    environmentInjector: EnvironmentInjector, configLoader: RouterConfigLoader,
```